challenges/odd-divisor/test/index.test.js

```diff
@@ -11,7 +11,16 @@ const tests = [
   [10, 36],
   [11, 47],
   [12, 50],
-  // [1000000000, 333333333334181300], // @TODO: Confirm if correct value
+  [13, 63],
+  [14, 70],
+  [15, 85],
+  [16, 86],
+  [17, 103],
+  [18, 112],
+  [100, 3344],
+  [200, 13344],
+  [1000, 333396],
+  [1000000000, 333333333334181300], // @TODO: Confirm if correct value
```

Comment:
There is a better way to test.
Try `sum(1000000000-1)-sum(1000000000-2)`, what's your result?

Reply:
Also I am pretty confident that the result should be `333333333334181226`